samples/nrf9160/modem_shell/prj.conf

```diff
@@ -30,8 +30,8 @@ CONFIG_SHELL=y
 CONFIG_SHELL_WILDCARD=n
 CONFIG_SHELL_PROMPT_UART="mosh:~$ "
 CONFIG_SHELL_ARGC_MAX=40
-# Command line buffer is set this large to enable writing of certificates.
-CONFIG_SHELL_CMD_BUFF_SIZE=3072
+# Command line buffer is set this large to enable injecting A-GPS data on command line.
```

Comment:
```suggestion
# Command line buffer is set this large to enable injecting A-GPS data on command line and writing of certificates
```
I'd like to keep certificates in the comment just that people are aware of the main reasons limiting decreasing of the shell command buffer.

Reply:
Ok, will change

Reply:
Yes, 17 comes from that exactly. However, there is already some slack in the 3500. A-GPS data is likely always smaller than 3500. But I can make it 3582, no problem